### ISSUE TITLE
CTD when playing Inferno

### ISSUE THREAD

**Issue by wang429** [dbjorge/jorbs-spire-mod#187]
Reported by Discord user Rindokisuto#8193
"In my current run I just got to the guardian - it doesn't matter when I play inferno, it always causes a crash"

Log from a reproduction by Rindokisuto#8193

```
Exception in thread "LWJGL Application" com.badlogic.gdx.utils.GdxRuntimeException: java.lang.NoSuchMethodError: stsjorbsmod.cards.wanderer.Inferno$1.addToBot(Lcom/megacrit/cardcrawl/actions/AbstractGameAction;)V
    at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:133)
Caused by: java.lang.NoSuchMethodError: stsjorbsmod.cards.wanderer.Inferno$1.addToBot(Lcom/megacrit/cardcrawl/actions/AbstractGameAction;)V
    at stsjorbsmod.cards.wanderer.Inferno$1.update(Inferno.java:50)
    at com.megacrit.cardcrawl.actions.GameActionManager.update(GameActionManager.java:155)
    at com.megacrit.cardcrawl.rooms.AbstractRoom.update(AbstractRoom.java:326)
    at com.megacrit.cardcrawl.dungeons.AbstractDungeon.update(AbstractDungeon.java:2494)
    at com.megacrit.cardcrawl.core.CardCrawlGame.update(CardCrawlGame.java:871)
    at com.megacrit.cardcrawl.core.CardCrawlGame.render(CardCrawlGame.java:422)
    at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:225)
    at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:126)
```

Seems like an issue with calling `addToBot()` in the anonymous class in Inferno.

maybe related to #186

**Comment by wang429** [comment]
This is an issue with running on main rather than beta.

**Comment by wang429** [comment]
fixed by #188
